packages/@adobe/spectrum-css-temp/components/slider/index.css

```diff
@@ -611,3 +611,22 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+.spectrum-Slider-controls {
+  &::before,
+  &::after {
+    content: '';
+    display: block;
+    inline-size: var(--spectrum-slider-controls-margin);
+    position: absolute;
+    block-size: 100%;
+  }
+  &::before {
+    inset-inline-start: calc(var(--spectrum-slider-controls-margin) * -1);
+  }
+  &::after {
+    inset-inline-end: calc(var(--spectrum-slider-controls-margin) * -1);
+  }
+}
```

Comment:
Noticed that this doesn't quite work with trying to click on the right most limit for sliders with side labels. I think it is because the label element overlaps the ::after?
![image](https://user-images.githubusercontent.com/9661127/102546451-8364d080-406c-11eb-9208-e5c2d4a68cc0.png)

Reply:
Ah, looks like the solution is to make that right label container have `margin-left` rather than `margin-right` (or w/e the RTL equivalent is)

Reply:
imma see about adding a new class for the value label container and give it the margin